data/base-picking.yaml

```diff
@@ -66,30 +89,113 @@ picking:
+    poisoncrossbow: concealing the points of several crossbow bolts glistening with moisture
+    reaper: crust-covered black scarab of some unidentifiable substance
+    scythe: you notice a glint of razor sharp steel hidden within a suspicious looking seam
+    shadowling: you notice a small black crystal deep in the shadows of the
+    shocker: You notice two silver studs right below the keyhole
+    shrapnel: keyhole is packed tightly with a powder around the insides of the lock
+    sleep: Two sets of six pinholes on either side of the
+    teleport: covered with a thin metal circle that has been lacquered with a shade
 
-  disarm_quick:
-  - trivially constructed gadget which you can take down any time
-  - This trap is a laughable matter
-  - An aged grandmother could defeat this trap in her sleep
+  disarmed_traps:
+    #acid:
+    boomer: separated harmlessly from their charge
+    #bouncer:
+    #concussion:
+    crossbow: have been bent in such a way that they no longer will function # poisoncrossbow
+    curse: You see a glowing rune pushed deep within the chest
+    cyanide: the dart has been moved too far out of position for the mechanism to function
+    disease: animal bladder and a disconnected string
+    fireants: indicating the trap is no longer a danger
+    fleas: have been bent away from each other
+    frog: It seems far enough away from the lock to be harmless
+    laughgas: You deem it quite safe
+    lightning: seems a small portion of the trap has been removed
+    mana: The seal has been pried away from the lid
+    mime: metallic visage rests a small deflated bladder
+    naphthasoak: indicating a liquid was drained out
+    naphthafire: as if something had been poured out the hole
+    #nervepoison:
+    poison: A bent needle sticks harmlessly out
+    poisongas: Someone has unhooked the stopper, rendering it harmless
+    reaper: was picked apart and removed from the
+    scythe: It is no longer attached to a razor-sharp scythe blade
+    shadowling: It seems harmless
+    shocker: whatever it was has been pried out
+    #shrapnel:
+    sleep: sealed with dirt, blocking whatever
+    teleport: has been peeled away from the hinges
 
-  disarm_normal:
-  - will be a simple matter for you to disarm
-  - You think this trap is precisely at your skill level
+  trap_parts:
+  - glass reservoir
+  - steel striker
+  - black cube
+  - chitinous leg
+  - spring
+  - brown clay
+  - animal bladder
+  - sharp blade
+  - tiny hammer
+  - sealed vial
+  - stoppered vial
+  - iron disc
+  - spring
+  - lever
+  - striker
+  - needle
+  - curved blade
+  - silver studs
+  - striker
+  - metal circle
+  - steel pin
+  - broken rune
+  - green runestone
+  - bronze seal
+  - glass sphere
+  - bronze face
+  - black crystal
+  - capillary tube
 
   lockpick_costs:
     # Riverhaven is not added because the lockpicks duplicate Crossing and mess things up. Please don't add it.
```

Comment:
I think to accommodate per city costs for similar named lockpicks, this data could be reorganized under "hometown" city names like:

```yaml
lockpick_costs:
  Crossing:
    ordinary: 125
    stout: 250
    ...
  Riverhaven:
    ordinary: xxx
    stout: yyy
    ...
  Shard:
    stout iron: 270
    ...
```

Then in `pick.lic` initialize method would fetch the lockpick costs like this:

```ruby
@lockpick_costs = picking_data['lockpick_costs'][@settings.hometown]
```
And that should be fine because when it comes time to actually go buy the lockpicks, that code already uses `@settings.hometown` in the refill_ring method:

```ruby
DRCT.refill_lockpick_container(@settings.lockpick_type, @settings.hometown, @lockpick_container, lockpicks_needed)
```

Reply:
Yep, this seems to work. I went up to Riverhaven and, with the above changes I proposed, ran `;pick refill` and the script took me to withdraw lirums and bought lockpicks per the Riverhaven price.

```
--- Lich: go2 has exited.

[pick]>buy ordinary lockpick

> 
Ss'Thran smiles, baring his fangs. "Ahh, the lockpick.  That'll cost you 100 lirums."
> 
[pick]>offer 100

Ss'Thran hands you an ordinary lockpick.

> 
[pick]>put my lockpick on my lockpick wristcuff

You put your lockpick on your lockpick wristcuff.
```

Reply:
I've got nothing against that change. Do you want me to make it as part of this, or do you want to add it after the fact?

Reply:
@wstampley Sorry, I didn't see your comment! I can submit a PR to your fork and you can decide if you want it with your rewrite or I can submit it separately afterwards to this repo. Thanks

Reply:
I've submitted https://github.com/wstampley/dr-scripts/pull/5 to your fork